src/NGen.cpp

```diff
@@ -539,6 +539,17 @@ int main(int argc, char *argv[]) {
 
     } //done time
 
+  //Close down any file handles we have
+  for(auto& f : nexus_outfiles){
+    f.second.close();
```

Comment:
Similar to below, `nexus_outfiles` should be moved into the function and placed within an RAII block, so the destructor can handle it.

Reply:
Nexus output handling is still pretty rough, there's been a long standing [TODO](https://github.com/NOAA-OWP/ngen/blob/52f43540239e202328c7c9350149f9f5b8f1f409/src/NGen.cpp#L425) to refactor it -- but its been pretty low priority.